tests/functional/xhr/ajax-events.test.js

```diff
@@ -149,7 +149,8 @@ testDriver.test('Distributed Tracing info is added to XHR ajax events', xhrBrows
         enabled: true
       },
       ajax: {
-        harvestTimeSeconds: 2
+        harvestTimeSeconds: 2,
+        enabled: true
```

Comment:
Any idea why this was not failing before?

Reply:
i wasn't there for the merge, but i had the same question.  thought it was maybe some sort of merge conflict where the local version didn't have the flag?